QUEST_LV_0100.tsv

```diff
@@ -1,124 +1,124 @@
+QUEST_LV_0100_20150317_000012	$Soldier Block
+QUEST_LV_0100_20150317_000013	$Be warned!{nl}Monsters are attacking, so all citizens must evacuate!{nl}But, why are those citizens are in the military zone?{nl}
+QUEST_LV_0100_20150317_000014	$Ah, so you are the Revelator! I'm sorry, I didn't recognize you.{nl}Monsters are attacking the supply base at the moment.{nl}I need to get some important rations out from the base. Please help me!
+QUEST_LV_0100_20150317_000015	$It was peaceful not too long ago, but now they are coming in groups.{nl}What's happening exactly?
+QUEST_LV_0100_20150317_000016	$Even so, I felt uneasy...{nl}Who knew that they'd attack in such an absurd manner.
+QUEST_LV_0100_20150317_000017	xx$The rations are in the First Supply Warehouse, which you can get to with the road on the upper side.{nl}Bring however little leftovers you can find... 
+QUEST_LV_0100_20150317_000018	$I wrote down who Sparnas is, maybe it will be useful in the future.{nl}Though I may not appear so right now, I was a biologist four years ago.
+QUEST_LV_0100_20150317_000019	$Please pass this report to Luders.{nl}He is probably at the Center Supply Camp.
+QUEST_LV_0100_20150317_000020	$Officer Luders
+QUEST_LV_0100_20150317_000021	Ah, you must be the so called Revelator. Nice to meet you.{nl}Is this Block's report?{nl}
+QUEST_LV_0100_20150317_000022	$Hmm... Is that so?{nl}They just destroyed the supplies and ran away. Even if Chupacabras had wings, I am sure they can't do something like this.{nl}
+QUEST_LV_0100_20150317_000023	$This isn't enough.{nl}I am sorry, but please bring us Moody's report from Amzina Curved Path.
+QUEST_LV_0100_20150317_000024	$I feel better since Luders is here.
+QUEST_LV_0100_20150317_000025	$Soldier York
+QUEST_LV_0100_20150317_000026	$Shaulems are crossing the road.{nl}I am in a panic myself, and I don't know what to do.{nl}I'm sure Luders won't allow this to happen... What should we do?
+QUEST_LV_0100_20150317_000027	$Do you think the problem will be resolved just by defeating Sparnas?
+QUEST_LV_0100_20150317_000028	$Thanks to your help, I think I can hold myself together.{nl}But... won't this kind of thing happen again? what should I do then...?
+QUEST_LV_0100_20150317_000029	$Sparnas? We don't have a clear answer about it.{nl}It just appeared from the sky without any signs and destroyed all the supplies we had. Then, it quickly disappeared.{nl}We got headaches just hearing its cries.
+QUEST_LV_0100_20150317_000030	$The place I just talked about is the Second Supply Warehouse.{nl}It is depressing to think how these kinds of things will be recurring in the future.
+QUEST_LV_0100_20150317_000031	$Take it. I made some bait from old grains in the supplies.{nl}I will lure it from up there, so you stay here.{nl}Sparnas will definitely appear at one of these two places. Don't let it escape, destroy it for sure.{nl}
```

Comment:
"Don't let it escape, destroy it for sure." -> "Don't let it escape. Destroy it for sure."